client/app/components/InputWithCopy.jsx

```diff
@@ -42,7 +43,10 @@ export default class InputWithCopy extends React.Component {
   render() {
     const copyButton = (
       <Tooltip title={this.state.copied || "Copy"}>
-        <CopyOutlinedIcon style={{ cursor: "pointer" }} onClick={this.copy} />
+        <PlainButton onClick={this.copy}>
+          {/* TODO: MISSING_VISUAL_FEEDBACK */}
```

Comment:
Can we link this to an issue or provide more explanation?

Reply:
We have the issue. The visual feedback here refers to focus/hover/active styles, which are lacking or inconsistent across the UI. [Tracker](https://databricks.atlassian.net/browse/REDASH-1102?atlOrigin=eyJpIjoiYTI1NWNjMTgyY2I2NGViMTk5MzMwMmRkZTk2MDA2ZGUiLCJwIjoiaiJ9)

Reply:
I keep my position against those TODO comments -> https://github.com/getredash/redash/pull/5424#discussion_r598001895